### ISSUE TITLE
Website Accessibility-WCAG 4.1.2: Nested interactive controls are not announced by screen readers (li[aria-controls="example-1-sass"])

### ISSUE THREAD

**Issue by A11Ybot-OP** [sass/sass#3232]
#### Issue
`Nested interactive controls are not announced by screen readers` ([nested-interactive](https://dequeuniversity.com/rules/axe/4.3/nested-interactive?application=msftAI))
#### Target application
[Sass: Sass Basics](https://sass-lang.com/guide)
#### Element path
li[aria-controls="example-1-sass"]
#### Snippet
```<li class="ui-tabs-tab ui-corner-top ui-state-default ui-tab" role="tab" tabindex="-1" aria-controls="example-1-sass" aria-labelledby="ui-id-2" aria-selected="false" aria-expanded="false"><a href="#example-1-sass" class="ui-tabs-anchor" role="presentation" tabindex="-1" id="ui-id-2">Sass</a></li>```
#### How to fix
    Fix any of the following:
      Element has focusable descendants
#### Environment
Microsoft Edge version 96.0.1054.62
